user_guide_src/source/intro/index.rst

```diff
@@ -17,9 +17,9 @@ or completely replaced to make the system work the way you need it to.
 In short, CodeIgniter is the malleable framework that tries to provide
 the tools you need while staying out of the way.
 
-***********************
-Who is CodeIgniter For?
-***********************
+************************
+For Whom is CodeIgniter?
```

Comment:
English is not my first language but that sounds really formal and a bit unnatural to me. Can we go back to the initial form?

Reply:
Considering the word order, the correction should have been: "Whom is CodeIgniter For?" But this construction is too informal.

The original construction of "Who ... for" is grammatically incorrect as "who" would be referring to CodeIgniter and "for" is misplaced. The intended usage of the title is asking for the intended users so it should have been "whom".

Reply:
I will have to leave this decision to @MGatner since I'm not qualified enough 😅

Reply:
The emendation is grammatically correct... and very stiff. But using "who" where one should use "whom" is a bit egregious (in my book). What if we ditch this and go with something else altogether, like "Is CodeIgniter For You?" or "CodeIgniter's Target Audience" or "Who Uses CodeIgniter?" - something both keeping with the User Guide's colloquial tone without butchering the already poor state of the English language.